src/components/Button/Button.scss

```diff
@@ -138,6 +138,10 @@ $stacking-order: (
   }
 }
 
+.Hidden {
+  visibility: hidden;
```

Comment:
At first I tried using the `visually-hidden` mixin, but this doesn't preserve the space that the icon is taking up, regardless if it is hidden or not.